src/base.ts

```diff
@@ -873,4 +873,13 @@ export default class KintoClientBase {
       )
     );
   }
+
+  /**
+   * Retrieves server endpoints.
+   *
+   * @return {Object}
```

Comment:
Since we have TypeScript return types, you can omit the JSDoc return line. The other functions that have this are leftover from before the TypeScript transition.